build/configs/artik055s/Make.defs

```diff
@@ -119,20 +123,20 @@ ifeq "$(ARCHMINOR)" "9"
 endif
 endif
 
-endif
-
 ARCHDEFINES =
 ARCHPICFLAGS = -fpic -msingle-pic-base -mpic-register=r10
 
 CFLAGS = $(ARCHCFLAGS) $(ARCHWARNINGS) $(ARCHOPTIMIZATION) $(ARCHCPUFLAGS) $(ARCHINCLUDES) $(ARCHDEFINES) $(EXTRADEFINES) -pipe -ffunction-sections -fdata-sections
 CPICFLAGS = $(ARCHPICFLAGS) $(CFLAGS)
-CXXFLAGS  = $(ARCHCXXFLAGS) $(ARCHWARNINGSXX) $(ARCHOPTIMIZATION) $(ARCHCPUFLAGS) $(ARCHXXINCLUDES)
-CXXFLAGS += $(ARCHDEFINES) $(EXTRADEFINES) -pipe -std=c++11 -DCLOCK_MONOTONIC -D__TINYARA__
-CXXFLAGS += -fno-exceptions -fcheck-new -fno-rtti
-CXXFLAGS += -pedantic -D_DEBUG -D_LIBCPP_BUILD_STATIC -D_LIBCPP_NO_EXCEPTIONS -ffunction-sections -fdata-sections
+CXXFLAGS = $(ARCHCXXFLAGS) $(ARCHWARNINGSXX) $(ARCHOPTIMIZATION) $(ARCHCPUFLAGS) $(ARCHXXINCLUDES) $(ARCHDEFINES) $(EXTRADEFINES) -pipe
 ifeq ($(CONFIG_LIBCXX),y)
 CXXFLAGS += -DCONFIG_WCHAR_BUILTIN
 endif
+ifeq ($(CONFIG_AUDIO),y)
+  CXXFLAGS += -std=c++11 -DCLOCK_MONOTONIC -D__TINYARA__
+  CXXFLAGS += -fno-exceptions -fcheck-new -fno-rtti
+  CXXFLAGS += -pedantic -D_DEBUG -D_LIBCPP_BUILD_STATIC -D_LIBCPP_NO_EXCEPTIONS -ffunction-sections -fdata-sections
```

Comment:
Is this for audio? or libcxx? Previous file is under another config. I think we should use the same config for this. I think libcxx is correct.